src/store/Branding.ts

```diff
@@ -84,9 +76,28 @@ export const actionCreators: ActionCreators = {
 
 };
 
+const getBrandingData = (receivedBranding?: { [key: string]: any }): BrandingData => {
```

Comment:
Is there any reason not to use `function` here?

Reply:
Let it be a  `function`.